app/goal_routes.py

```diff
@@ -0,0 +1,94 @@
+from flask import Blueprint, jsonify, abort, make_response, request
```

Comment:
👍🏾 These imports help a great deal when it comes to our request & response cycle